### PR TITLE
add Vietnamese translation

### PR DESCRIPTION
**Pull request by quanengineering**
## Motivation

I added the Vietnamese translation. Feel free to correct any mistakes.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### CODE REVIEW

File: packages/docusaurus-theme-classic/codeTranslations/vi.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+  "theme.AnnouncementBar.closeButtonAriaLabel": "Đóng",
+  "theme.CodeBlock.copied": "Đã sao chép",
+  "theme.CodeBlock.copy": "Sao chép",
+  "theme.CodeBlock.copyButtonAriaLabel": "Sao chép code vào bộ nhớ tạm (clipboard)",
```
Comment (lex111) [inline packages/docusaurus-theme-classic/codeTranslations/vi.json:5]: > clipboard

@quanengineering it is strictly necessary to specify the original word in parentheses or we can avoid it for brevity?

Comment (quanengineering) [reply packages/docusaurus-theme-classic/codeTranslations/vi.json:5]: At first, I think some users may find it difficult to understand some technical terms so I keep original word in parentheses in some place. For now, I think most users will understand, so I will remove original words.

Comment (quanengineering) [reply packages/docusaurus-theme-classic/codeTranslations/vi.json]: @lex111 I have just removed all original word.

**Review by lex111 — APPROVED** [review]

**Comment by lex111** [comment]
@quanengineering seems good, thanks!

**Pull request merged**